localstack/services/ses/provider.py

```diff
@@ -149,7 +148,23 @@ def register_ses_api_resource():
     global _EMAILS_ENDPOINT_REGISTERED
 
     if not _EMAILS_ENDPOINT_REGISTERED:
-        get_internal_apis().add(SesServiceApiResource())
+        ses_service_api_resource = SesServiceApiResource()
+        get_internal_apis().add(
+            Resource(
+                "/_localstack/ses",
+                DeprecatedResource(
+                    ses_service_api_resource,
+                    previous_path="/_localstack/ses",
+                    deprecation_version="1.4.0",
+                    new_path="/_aws/ses/",
+                ),
+            )
+        )
+
+        from localstack.services.edge import ROUTER
+
+        ROUTER.add(Resource("/_aws/ses/", ses_service_api_resource))
```

Comment:
Maybe use `EMAILS_ENDPOINT` (not sure if trailing slash is required)

```suggestion
        ROUTER.add(Resource(f"{EMAILS_ENDPOINT}/", ses_service_api_resource))
```